scripts/bump-version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Should this script also address the `dependency_overrides` problem?
To be able to release a version, These lines https://github.com/getsentry/sentry-dart/blob/main/flutter/pubspec.yaml#L29-L31 should be commented out, and after release (Maybe via post-release script?) should be back again.

Reply:
Added. Although its "meh" solution, as it will always append removed `dependencies_override` to the end of the file. Shouldn't be too bad though.

Reply:
yeah unfortunately I didn't find any way around `dependencies_override`, either we comment it out and comment it back before/after release, or we lose the ability to debug all the packages together.